__e2e__/init.test.ts

```diff
@@ -193,3 +193,17 @@ test('init --platform-name should work for out of tree platform', () => {
 
   expect(dirFiles.length).toBeGreaterThan(0);
 });
+
+test('init should contain Cocoapods instructions when pods are not installed automatically', () => {
+  createCustomTemplateFiles();
+
+  const {stdout} = runCLI(DIR, [
+    'init',
+    PROJECT_NAME,
+    '--skip-install',
+    '--install-pods',
```

Comment:
i think `--install-pods` is not necessary here as `--skip-install` won't install it anyway